README.md

```diff
@@ -50,24 +48,33 @@ This will ensure any dependencies are ready before launching Xcode.
 
 You can also open the project by double clicking on Simplenote.xcworkspace file, or launching Xcode and choose `File` > `Open` and browse to `Simplenote.xcworkspace`.
 
+Once you have opened Simpleonte iOS in Xcode, depending on your setup, you may need to make a few changes before you can build the app.  In Xcode hit `Command + B` and see if you get any errors.  
+
+If you see `The server SSH fingerprint failed to verify` before you can build Simplenote you will need to mark the app as trusted.  To do this, tap on the warning and hit Trust
```

Comment:
Where would this error show up? In the device itself?

Reply:
It shows up in Xcode if you have not verified the project

Reply:
And then stops you from building the app